core/trino-main/src/main/java/io/trino/execution/SqlTaskExecutionFactory.java

```diff
@@ -44,6 +45,9 @@ public class SqlTaskExecutionFactory
     private final Tracer tracer;
     private final boolean perOperatorCpuTimerEnabled;
     private final boolean cpuTimerEnabled;
+    // SqlTaskManager is responsible for moving task from INITIALIZE -> RUNNING.
+    // When using this factory without SqlTaskManager, set to true
```

Comment:
`When using this factory without SqlTaskManager`

Is it only for testing? add a comment

Reply:
Correct. Will do.